test/e2e/apibinding/apibinding_test.go

```diff
@@ -108,7 +118,7 @@ func TestAPIBinding(t *testing.T) {
 		require.NoError(t, err)
 
 		uniqueExpectedURLSet := sets.NewString()
```

Comment:
nit: the logic for this unique set is now redundant given that you've updated the input to be unique as well

Reply:
done